src/pages/geophysics/components/map-container.tsx

```diff
@@ -0,0 +1,43 @@
+import { MapWidgets } from '@/pages/wetlands/components/map-widgets';
+import { MapContextMenu } from "@/components/custom/map/map-context-menu";
+import { PopupDrawer } from "@/components/custom/popups/popup-drawer";
+import { useMapContainer } from "@/hooks/use-map-container";
+import { PROD_GEOSERVER_URL } from '@/lib/constants';
+
+export default function ArcGISMap() {
+    const {
+        mapRef,
+        contextMenuTriggerRef,
+        drawerTriggerRef,
+        popupContainer,
+        setPopupContainer,
+        popupContent,
+        clickOrDragHandlers,
+        handleOnContextMenu,
+        coordinates,
+        setCoordinates,
+    } = useMapContainer({
+        wmsUrl: `${PROD_GEOSERVER_URL}wms`
```

Comment:
this might not be apparent yet but we will eventually need to add a layerConfig to this hook argument. it can be done later once we start adding more functionality.